internal/tools/pipeline/aop/plugins/pipeline/basic/plugin.go

```diff
@@ -76,10 +80,16 @@ func (p *provider) Handle(ctx *aoptypes.TuneContext) error {
 		content.TaskInfos = append(content.TaskInfos, taskReport)
 	}
 
-	_, err = ctx.SDK.Report.Create(apistructs.PipelineReportCreateRequest{
+	var reqMeta map[string]interface{}
+	b, _ := json.Marshal(map[string]interface{}{
+		"data": content,
+	})
+	_ = json.Unmarshal(b, &reqMeta)
+	pbMeta, err := structpb.NewStruct(reqMeta)
```

Comment:
err not handled

Reply:
![image](https://user-images.githubusercontent.com/30427474/177686153-848b0169-2830-47b0-a2e1-12788341d4ae.png)

report provider unified convert to pb meta